python3.8/cookiecutter-aws-sam-efs-python/{{cookiecutter.project_name}}/tests/integration/test_api_gateway.py

```diff
@@ -0,0 +1,41 @@
+import os
+from unittest import TestCase
+
+import boto3
+import requests
+
+"""
+Make sure env variable AWS_SAM_STACK_NAME exists with the name of the stack we are going to test. 
+"""
+
+
+class TestApiGateway(TestCase):
+    api_endpoint: str
+
+    def setUp(self) -> None:
+        """
+        Based on the provided env variable AWS_SAM_STACK_NAME,
+        here we use cloudformation API to find out what the HelloEfsApi URL is
+        """
+        stack_name = os.environ.get("AWS_SAM_STACK_NAME")
+        self.failIf(not stack_name, "Cannot find env var AWS_SAM_STACK_NAME")
```

Comment:
These errors are helpful, but given that this is a template integration tests, probably suggestion solution to these errors in error message as well. Eg: `Cannot find env var AWS_SAM_STACK_NAME, please setup this environment variable with the stack name where we are running integration tests.`.